### PR TITLE
Run makefile command fmt and goimports with test command

### PR DESCRIPTION
**Pull request by LinuxSuRen**
### What type of PR is this?

/kind cleanup

### What this PR does / why we need it:
it is very easy to forget the format command after some code lines changed. but the comand is necessary for all contributors, it is not easy to forget

### Which issue(s) this PR fixes:

None existing issues.

### Special notes for reviewers:
```
None
```

### Does this PR introduced a user-facing change?

```release-note
None
```

### Additional documentation, usage docs, etc.:

```docs
None
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -133,7 +133,7 @@ helm-uninstall:
 	kubectl delete -f https://raw.githubusercontent.com/kubesphere/ks-installer/master/roles/ks-core/prepare/files/ks-init/role-templates.yaml
 
 # Run tests
-test: vet
+test: vet fmt goimports
```
Comment (zryfish) [inline Makefile:136]: I have a second thought, we use `make test` in Prow, adding `fmt goimports` will slow our pipeline. We have presubmit checks to find those unformatted PRs. For developers, `make verify-all` before submitting is a good practice.

Comment (LinuxSuRen) [reply Makefile:136]: `make verify-all` is a definitely good solution to do the check work. But this is my first time to know that makefile command. It's quite important that letting contributors know fewer things before they get started their contributions. How about the following solution:

```
test: vet verify-all
  xxx

test-without-verify: vet
  xxx
```

then let Prow runs the command `make test-without-verify`.

Comment (zryfish) [reply Makefile:136]: I think it's better to make it more clear that contributors should run `make verify-all` before submitting in our development guide. Also, if they don't run `make verify-all`, our presubmit will complain and they will do it next time :)

Comment (LinuxSuRen) [reply Makefile:136]: Ok than. But I still believe that the process should be as easy as possible. I'm going to close this PR until others want to simplify the process.

**Pull request closed without merging**